dev/scripts/server545_puppet3_compatibility.sh

```diff
@@ -5,7 +5,7 @@
 
 
 # Puppet Server package version (from http://builds.puppetlabs.lan/puppetserver/)
-export PACKAGE_BUILD_VERSION='2.1.0.SNAPSHOT.2015.04.17T0057'
+export PACKAGE_BUILD_VERSION='2.1.0.SNAPSHOT.2015.05.14T1222'
```

Comment:
This comment is directly about the changes in this PR, but it seems a bit weird to me that we have a hard-coded version here ... shouldn't the beaker use the version of the package was built earlier in the pipeline?

Or ... I see this is under `dev/scripts` ... I guess it's not part of the acceptance test suite?